libs/hdf-converters/src/neuvector-mapper.ts

```diff
@@ -0,0 +1,311 @@
+              score: {path: 'score'},
+              vectors: {path: 'vectors'},
+              // In the neuvector/scanner dashboard, the CVSS v2 and v3 scores are selectable by a dropdown.
+              vectors_v3: {path: 'vectors_v3'},
+              score_v3: {path: 'score_v3'},
+              // Appears to be CVSS v3, since info for v2 and v4 doesn't always exist for CVEs.
+              severity: {path: 'severity'},
+              source: {
+                path: 'package_name',
+                transformer: (packageName: string) =>
+                  this.getModule(packageName)?.source
+              },
+              status: {
+                path: 'name',
+                transformer: (name: string) =>
+                  this.rawData.report.modules
+                    ?.find((module) => module.cves?.find(cveIdMatches(name)))
+                    ?.cves?.find(cveIdMatches(name))?.status
+              },
+              feed_rating: {path: 'feed_rating'},
+              link: {path: 'link'},
+              // Both fields are Unix epoch timestamps, in seconds.
+              published_timestamp: {path: 'published_timestamp'},
+              last_modified_timestamp: {path: 'last_modified_timestamp'},
+              ...universalTags()
+            },
+            refs: [],
+            source_location: {ref: {path: 'file_name'}},
+            id: {
+              transformer: (data: Record<string, any>) =>
+                `${data.name}/${data.package_name}/${data.package_version}`
+            },
+            desc: {path: 'description'},
+            impact: {
+              transformer: (data: Record<string, any>) => {
+                const score = data.score_v3 ?? data.score;
+                return Number((score / 10).toFixed(1));
+              }
+            },
+            code: null,
+            results: [
+              {
+                status: ExecJSON.ControlResultStatus.Failed,
+                code_desc: '',
+                message: {
+                  transformer: (data: Record<string, any>) => {
+                    const {package_name, package_version, fixed_version} = data;
+                    if (!fixed_version) {
+                      return `Vulnerable package ${package_name} is at version ${package_version}. No fixed version available.`;
+                    }
+                    return `Vulnerable package ${package_name} is at version ${package_version}. Update to fixed version ${fixed_version}.`;
+                  }
+                },
+                start_time: ''
+              }
+            ]
+          }
+        ],
+        sha256: ''
+      }
+    ],
+    passthrough: {
+      transformer: (data: Record<string, any>): Record<string, unknown> => {
+        return {
+          auxiliary_data: [{name: '', data: _.omit([])}], //Insert service name and mapped fields to be removed
```

Comment:
Don't forget to fill this out.  We're only going to omit report.vulnerabilities and whatever report.whatevers gets mapped such as potentially the target information since we're going to use that in the profile.title.